controllers/kustomization_controller.go

```diff
@@ -65,6 +65,7 @@ import (
 	sourcev1 "github.com/fluxcd/source-controller/api/v1beta2"
 
 	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta2"
+	fluxClient "github.com/fluxcd/pkg/runtime/client"
```

Comment:
Please move this in the fluxcd/pkg section and rename the alias to runtimeClient